NEWS.md

```diff
@@ -1,5 +1,14 @@
 # dbplyr 1.0.0.9000
 
+## Backends
```

Comment:
I add the second level headings just before release (because it's too hard to know in advance exactly what will need a heading)